src/annotations/api/index.ts

```diff
@@ -41,9 +41,24 @@ export const writeAnnotation = async (
   ]
 }
 
+export const getAnnotations = async (
+  stream?: string
+): Promise<AnnotationStream[]> => {
+  const res = await axios.get(`${url}?${formatAnnotationQueryString({stream})}`)
+
+  if (res.status >= 300) {
+    throw new Error(res.data?.message)
+  }
+
+  return res.data.map((retrievedAnnotation: AnnotationStream) => ({
+    stream: retrievedAnnotation.stream,
+    annotations: retrievedAnnotation.annotations,
+  }))
+}
```

Comment:
we'll want to figure out the proper API here vis-a-vis `getAnnotation` and `getAnnotations`